src/library/DIPS.Mobile.UI/Components/Pickers/Platforms/iOS/UIDateTimePickerViewController.cs

```diff
@@ -0,0 +1,151 @@
+using CoreGraphics;
+using DIPS.Mobile.UI.Platforms.iOS;
+using DIPS.Mobile.UI.Resources.Colors;
+using DIPS.Mobile.UI.Resources.LocalizedStrings.LocalizedStrings;
+using DIPS.Mobile.UI.Sizes.Sizes;
+using Foundation;
+using Microsoft.Maui.Platform;
+using UIKit;
+using Colors = DIPS.Mobile.UI.Resources.Colors.Colors;
+
+namespace DIPS.Mobile.UI.Components.Pickers.Platforms.iOS;
+
+public abstract class UIDateTimePickerViewController : UIViewController
+{
+    private readonly IDateTimePicker m_dateTimePicker;
+    private UIButton? m_doneButton;
+    private UILabel? m_description;
+    
+    private UIStackView m_content;
+    private UIButton? m_closeButton;
+    private UIStackView? m_toolbarStackLayout;
+    private UIDatePicker? m_UIDateTimePicker;
+
+    public UIDateTimePickerViewController(IDateTimePicker dateTimePicker)
+    {
+        m_dateTimePicker = dateTimePicker;
+    }
+    
+    public override void ViewDidLoad()
+    {
+        m_content = new UIStackView { Axis = UILayoutConstraintAxis.Vertical, Distribution = UIStackViewDistribution.EqualSpacing};
```

Comment:
Brukes dette nå?

Reply:
Altså, setter vi content selv lenger? Trodde alt var innebygd i at vi bruker compact / in line nå?